float64 thrust
float64 tau_x
float64 tau_y
float64 tau_z

